src/main/java/com/revature/controllers/QuestionEvalController.java

```diff
@@ -0,0 +1,98 @@
+package com.revature.controllers;
+
+import com.revature.domain.QuestionComment;
+import com.revature.domain.QuestionEval;
+import com.revature.services.QuestionLogic;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value="/api/v1/evaluations/{evalId}/questions")
+public class QuestionEvalController {
+	
+	@Autowired
+	QuestionLogic questionLogic;
+	
+//QUESTIONCOMMENT CUD---------------------------------
+	
+	@RequestMapping(method = RequestMethod.POST, value = "{questionId}/comments")
+	public ResponseEntity<QuestionComment> addQuestionComment(@RequestBody QuestionComment comment, @PathVariable ("questionId") Integer questionId){
+		return ResponseEntity.ok(questionLogic.createComment(comment,questionId));
+	}
+	
+	@RequestMapping(method = RequestMethod.PUT, value = "{questionId}/comments/{id}")
+	public ResponseEntity<QuestionComment> updateQuestionComment(@RequestBody QuestionComment comment, @PathVariable ("id") Integer id){
+		
+		QuestionComment currComment = questionLogic.getCommentById(id);
+		
+		if(currComment == null){
+			System.out.println("Comment with id " + id + " not found");
+	        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+		}
+		
+		if(comment.getCommentText() != null){
+			currComment.setCommentText(comment.getCommentText());
+		}
+		if(comment.getQuestionEval() != null){
+			currComment.setQuestionEval(comment.getQuestionEval());
+		}
+		
+		return ResponseEntity.ok(questionLogic.updateComment(currComment));
+	}
+	
+	@RequestMapping(method = RequestMethod.DELETE, value = "{questionId}/comments/{id}")
+	public ResponseEntity<QuestionComment> deleteQuestionComment(@PathVariable ("id") int id){
+		
+		QuestionComment currComment = questionLogic.getCommentById(id);
+		
+		if(currComment == null){
+			System.out.println("Comment with id " + id + " not found");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)